pom.xml

```diff
@@ -11,7 +11,8 @@
   <version>1.11.2-1-SNAPSHOT</version>
   <packaging>hpi</packaging>
   <name>Jenkins jQuery plugin</name>
-  <url>http://wiki.jenkins-ci.org/display/JENKINS/jQuery+Plugin</url>
+  <description>This plugin is a library plugin for other plugins to share common jQuery. It also allows users to use jQuery on each view descriptions.</description>
```

Comment:
I think it was decided some time ago to use X plugin instead of Jenkins X plugin. For a new plugin I will use X plugin, but I am also not sure what to do about old plugins. I have a few ones that I maintain, where I use the old Jenkins X plugin pattern. Maybe worth pinging someone in the IRC channel or just dropping a message to the mailing list; only in case someone wants to have a proper way forward :+1: